src/python/espressomd/lees_edwards.py

```diff
@@ -22,9 +22,23 @@
 @script_interface_register
 class LeesEdwards(ScriptInterfaceHelper):
 
-    """Interface to the Lees-Edwards boundary conditions.
+    """
+    Interface to the :ref:`Lees-Edwards boundary conditions`.
 
-       See documentation.
+    Attributes
+    ----------
+    protocol : :obj:`object`
+        Lees--Edwards protocol.
```

Comment:
where does the double-dash in `Lees--Edwards` come from?

Reply:
It's reStructuredText syntax to generate an en-dash symbol instead of a hyphen. When a method is named after multiple individuals, the en-dash is required: [Lees–Edwards](https://link.springer.com/article/10.1023/A:1014595628808), [Shan–Chen](https://www.sciencedirect.com/science/article/abs/pii/S0301932219309073), [Weeks–Chandler–Andersen](https://pubs.acs.org/doi/abs/10.1021/jp013579b).